.env.template

```
@@ -1,64 +1,70 @@
-# For local dev: use localhost with mapped port (7688)
-NEO4J_URI=bolt://neo4j:7687
+# --- Project Identity ---
+L9_PROJECT=l9-engine
+L9_ENV=dev
+
+# --- Neo4j ---
+NEO4J_URI=bolt://localhost:7687
 NEO4J_USERNAME=neo4j
-NEO4J_PASSWORD=l9-dev-password
-NEO4J_POOL_SIZE=20
+NEO4J_PASSWORD=change-me-in-production
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEO4J_PASSWORD key should go before the NEO4J_URI key

(UnorderedKey)